### PR TITLE
HTCONDOR-2262 use-new-router-syntax

### PR DESCRIPTION
**Pull request by JaimeFrey**
Old syntax is kept, commented out with text about how to enable it.

### CODE REVIEW

**Comment by timtheisen** [comment]
Once this is reviewed, we will release a HTCondor-CE 23.x version that matches the current HTCondor 23.x feature version.

File: config/02-ce-bosco.conf [hunk 1]
```diff
@@ -8,6 +8,13 @@
 
 # Basic route for submitting to BOSCO
 # Use osg-configure to set BOSCO_RMS and BOSCO_ENDPOINT
+JOB_ROUTER_ROUTE_SSH_Endpoint @=jrt
+  TargetUniverse = 9
+  GridResource = "batch $(BOSCO_RMS) $(BOSCO_ENDPOINT)"
+@jrt
+
+JOB_ROUTER_ROUTE_NAMES = SSH_Endpoint
+
```
Comment (ColeBollig) [inline config/02-ce-bosco.conf:17]: We should remove the deprecation warning from this config file too

File: config/02-ce-sge.conf [hunk 2]
```diff
@@ -7,6 +7,13 @@
 ###############################################################################
 
 # Basic route for submitting to SGE
+JOB_ROUTER_ROUTE_SGE_Cluster @=jrt
+  TargetUniverse = 9
+  GridResource = "batch sge"
+@jrt
+
+JOB_ROUTER_ROUTE_NAMES = SGE_Cluster
+
```
Comment (ColeBollig) [inline config/02-ce-sge.conf:16]: Should we remove the deprecation warning from this config file? Willing to keep in given the commented out old syntax configuration

**Review by ColeBollig — APPROVED** [review]
LGTM

**Pull request merged**